tests/src/StringValidatorFillerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Phramework\ValidateFiller;
+
+use PHPUnit\Framework\TestCase;
+use Phramework\Validate\ArrayValidator;
+use Phramework\Validate\EnumValidator;
+use Phramework\Validate\StringValidator;
+use Phramework\Validate\IntegerValidator;
+use Phramework\Validate\NumberValidator;
+
+/**
+ * @license https://www.apache.org/licenses/LICENSE-2.0 Apache-2.0
+ * @author Vasilis Manolas <vasileiosmanolas@gmail.com>
+ * @coversDefaultClass Phramework\ValidateFiller\StringValidatorFiller
+ */
+class StringValidatorFillerTest extends TestCase
+{
+    public function testFillReturnValueIsAString()
+    {
+        $validator = new StringValidator();
+
+        /**
+         * @var string
+         */
+        $returnedString = (new StringValidatorFiller())
```

Comment:
I have updated this. But because I used `git commit --amend`, github cannot recognize the previous commit.